### ISSUE TITLE
Android new device transfer, internal photo identifcation issue until storage clear

### ISSUE THREAD

**Issue by Pleune** [immich-app/immich#14432]
### The bug

I just got a new phone, and let android auto transfer all of my apps and settings over. Immich transferred, and I was already logged in. However, when I took new photos, and looked at them in the "On this device" menu, they show as the wrong image! The count is right, but the first photos I took on my new phone shows as the first photo I took on my old phone, and so on... Clearing the storage/cache fixed this after logging in the next time.

This also affected the image uploading. Since my new phone saw every picture I took as some other already-existing image, nothing would backup. It would show as all the images had already been uploaded.

Unfortunately I don't have any screenshots of this as I can't really recreate it again without resetting my phone.

I am assuming some device uid or something of that nature got copied over when it shouldn't have. I would recommend just disabling this android transfer (I believe the process is opt-out per app).

### The OS that Immich Server is running on

NixOS

### Version of Immich Server

v1.120.0

### Version of Immich Mobile App

v1.120.0 build.168

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
none
```

### Your .env content

```Shell
none
```

### Reproduction steps

1. Have old phone with immich app
2. Setup new phone with android "magic app transfer"
3. Take a picture and attempt to auto upload images

### Relevant log output

```shell
none
```

**Comment by alextran1502** [comment]
This is not Immich bug, I would say, because the internal asset id has been changed when using the "magic app transfer" You can fix this by either reinstalling the app or clearing the app cache/internal storage

**Comment by Pleune** [comment]
While I understand how this happens, and how it's an annoying one to have to address. However I would strongly argue to do something to prevent this from happening, as it's quite confusing to come across.

I know Android Restore Credentials is new, and I would argue that Immich's android all should just opt out of that process. Whole it would be nice for it to work, it is not at all a big deal for an easy solution.

I am only assuming that this could be a possible fix as the blog here mentions that apps can opt out of the process.
https://android-developers.googleblog.com/2024/11/maintain-strong-user-relationships-with-restore-credentials.html?m=1

**Comment by alextran1502** [comment]
Ah thank you for the pointer

**Comment by contagon** [comment]
Sorry to comment on a closed issue, I just happened to hit this on a new phone as well. New images were uploaded (as in showed the correct photo), but in my case the metadata showed my old phone metadata and the image dates were set to about when I got my old phone.

I figured out pretty quickly clearing storage/cache fixed this, but a non-technical user definitely wouldn't. I almost didn't even catch it was happening as it was kinda subtle. I'd recommend opting out as mentioned above, as it can cause some strange behavior which would be hard to catch for many.
